### ISSUE TITLE
Negative Delay

### ISSUE THREAD

**Issue by ok-ttc** [polywock/globalSpeed#372]
Hi there,

I am amazed by this extension. I have it because there is a noticeable delay with some playback, especially when I use an external Bluetooth device. I would need to add a negative delay. I don't know if this is technically possible, but the extension doesn't allow it at the moment.

Can this be implemented?

**Comment by polywock** [comment]
Hello. There's an [extension](https://chrome.google.com/webstore/detail/youtube-audiovideo-sync/mknmhikmjljhpccebpnplhicmcfjkgbk) for Youtube that may be of help. It mutes the Youtube video, and loads a separate audio only stream with the time position shifted. This is the only approach I found that works so far, but it's hard or even impossible to extend to other sites. To implement this on other websites, the following conditions need to met. 

1. The website must be able to serve audio only streams. Youtube does this, as they load the video and audio separately. Very few streaming websites do this like Youtube. 

2. The way  to grab the audio only stream must be understood. Either through reverse engineering it yourself, or looking online to see if anyone else managed to figure out how. For Youtube, some smart people already figured out how to do this. 

Short version: It's a lot of work to extend this approach to other websites.

**Comment by herocoder14** [comment]
> Hello. There's an [extension](https://chrome.google.com/webstore/detail/youtube-audiovideo-sync/mknmhikmjljhpccebpnplhicmcfjkgbk) for Youtube that may be of help. It mutes the Youtube video, and loads a separate audio only stream with the time position shifted. This is the only approach I found that works so far, but it's hard or even impossible to extend to other sites. To implement this on other websites, the following conditions need to met.
> 
> 1. The website must be able to serve audio only streams. Youtube does this, as they load the video and audio separately. Very few streaming websites do this like Youtube.
> 2. The way  to grab the audio only stream must be understood. Either through reverse engineering it yourself, or looking online to see if anyone else managed to figure out how. For Youtube, some smart people already figured out how to do this.
> 
> Short version: It's a lot of work to extend this approach to other websites.

Can you tell the name of extension as it is showing 404 error
